static/templates/admin_user_group_list.handlebars

```diff
@@ -1,10 +1,11 @@
 {{#with user_group}}
 <div class="user-group white-box" id="{{id}}">
     <div class="alert user-group-status"></div>
-    <h4>
-        <span class="name" data-placeholder="{{t 'Name' }}" contenteditable="true" spellcheck="false">{{name}}</span>
-        <span class="spacer">—</span>
-        <span class="description" data-placeholder="{{t 'Description' }}" contenteditable="true">{{description}}</span>
+    <h4 class="stream-name">
```

Comment:
`stream-name` is not a good class name here. This element is for user groups, so changing it to `user-group-name` more make sense.